src/psyclone/psyGen.py

```diff
@@ -1146,7 +1149,16 @@ def view(self):
         raise NotImplementedError("BaseClass of a Node must implement the "
                                   "view method")
 
-    def indent(self, count, indent="    "):
+    @staticmethod
+    def indent(count, indent=INDENTATION_STRING):
+        '''
+        Helper function to produce indentation strings.
+
+        :param int count: Number of indentation levels.
+        :param str indent: String representing one indentation level.
+        :return: Complete indentation string.
+        :rtype: str
+        '''
         result = ""
         for i in range(count):
```

Comment:
You can just do `return count*indent` (I think)

Reply:
Yep, much nicer